t/op/length.t

```diff
@@ -172,8 +172,8 @@ $SIG{__WARN__} = sub {
 };
 
 is(length(undef), undef, "Length of literal undef");
+undef $u;
```

Comment:
We can view this as a behavior change, renaming the variable would have probably a better choice in this case, no?

Reply:
Okay.

> We can view this as a behavior change, renaming the variable would have probably a better choice in this case, no?

Okay.  Changing.